#
# Config shared by all vehicles.
#

control_node:
  enable_console_output: true
  enable_image_views: false
  queue_sizes:
    sub_usb_cam_image_raw: 1
    sub_lrc2xav_msg: 1
    sub_raw_obstacles: 1
    pub_xav2lrc_msg: 1

lane_detector:
  init_steer_angle: 0.0
  eL_height: 0.2
  e1_height: 1.2563
  trust_height: 0.6667
  lp: 609.3
  dynamic_roi: true
  threshold: 80

lrc:
  log_path: /home/jetson/catkin_ws/logfiles/
  epsilon: 0.7
  lu_ob_A: 0.6817
  lu_ob_B: 0.3183
  lu_ob_L: 0.2817
  enable_console_output: true

  xavier_to_lrc:
    topic: /xav2lrc_msg
    queue_size: 1
  ocr_to_lrc:
    topic: /ocr2lrc_msg
    queue_size: 1
  lrc_to_xavier:
    topic: /lrc2xav_msg
    queue_size: 1
  lrc_to_ocr:
    topic: /lrc2ocr_msg
    queue_size: 1